tests/webTables.spec.ts

```diff
@@ -55,10 +45,9 @@ test.describe('Web Tables', () => {
     test('Test Case 4: Validate phone number and pet name on the Owner Information page ', async ({ page }) => {
         await page.getByRole('button', { name: 'Owners' }).click()
         await page.getByRole('link', { name: 'Search' }).click()
-        const phoneNumber = '6085552765'
-        const petName = await page.getByRole('row', { name: '2387 S. Fair Way' }).locator('td').nth(4).textContent()
-        await page.getByRole('row', { name: '2387 S. Fair Way' }).getByRole('link', { name: 'Peter McTavish' }).click()
-        await expect(page.getByRole('row', { name: 'Telephone' }).getByRole('cell').last()).toHaveText(phoneNumber)
+        const petName = await page.getByRole('row', { name: '6085552765' }).locator('td').nth(4).textContent()
+        await page.getByRole('row', { name: '6085552765' }).getByRole('link', { name: 'Peter McTavish' }).click()
```

Comment:
This locator can be simplified, as the name for the owner is unique.
`await page.getByRole('link', { name: 'Peter McTavish' }).click()`

Reply:
yes, you are right. Thank you!